js/sidemenu.js

```diff
@@ -0,0 +1,12 @@
+let openSideMenu = document.querySelector('.fa-bars');
```

Comment:
`querySelector` only selects the first one named `.fa-bars` - just want to confirm that we won't reuse that class somewhere else.